src/database/entitySpec.js

```diff
@@ -0,0 +1,59 @@
+const databaseStructure = {
+/** Users */
+  Users: {
+    id: Integer,
+    firstName: String,
+    lastName: String,
+    email: String,
+    password: String,
+    gender: String,
+    role: String, // portfolio in the company
```

Comment:
This should be roleId and should be an Integer. I raised this during the meeting we had

Reply:
It has been updated